data/catalog.yaml

```
@@ -735,3 +735,38 @@ works:
       "Related to situated software ([<a
       href='https://www.gwern.net/docs/technology/2004-03-30-shirky-situatedsoftware.html'>1</a>][<a
       href='https://buttondown.email/hillelwayne/archive/situated-software'>2</a>])"
+armories:
```

I noticed this entry didn't show up in the preview, and I believe it's this missing indentation. I'm starting to regret using a YAML here... 😓 

```suggestion
  armories:
```

---

Thank you! I was wondering what was wrong..